Whiskey/Tasks/Invoke-WhiskeyNUnit3Task.ps1

```diff
@@ -16,7 +16,19 @@ function Invoke-WhiskeyNUnit3Task
     Set-StrictMode -Version 'Latest'
     Use-CallerPreference -Cmdlet $PSCmdlet -SessionState $ExecutionContext.SessionState
 
-    $nunitPackage = 'NUnit.ConsoleRunner'
+    $nunitPackages = @('NUnit.ConsoleRunner')
+    if ($TaskParameter['SupportNUnit2'])
+    {
+        # NUnit.Console includes NUnit.ConsoleRunner and the NUnit2 Compatibility Extension(s)

```

Comment:
Can you re-write this so it's a little clearer? I'm having a hard time following along.